evaluate2.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+#
+#  Copyright 2023 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+set -eo pipefail
+
+if [ -z "$1" ]
+  then
+    echo "Usage: evaluate2.sh <fork name> (<fork name 2> ...)"
+    echo " for each fork, there must be a 'prepare_<fork name>.sh' script and a 'calculate_average_<fork name>.sh' script"
+    echo " there may be an 'additional_build_steps_<fork name>.sh' script too"
```

Comment:
Why do we need `additional_build_steps_*.sh`? I think the stuff could be in `prepare_*.sh`

They run one after another and there is a single use only:
* https://github.com/gunnarmorling/1brc/blob/main/additional_build_steps_thomaswue.sh
* https://github.com/gunnarmorling/1brc/blob/main/prepare_thomaswue.sh

I don't see why we can not `mv additional_build_steps_thomaswue.sh prepare_thomaswue.sh` and drop support for `additional_build_steps_*.sh`.